File: net/management-api-v2/GetValidationIssues.cs
// DocSection: mapi_v2_get_validation_issues
// Tip: Find more about .NET SDKs at https://kontent.ai/learn/net
using Kentico.Kontent.Management;

var client = new ManagementClient(new ManagementOptions
{
    ApiKey = "<YOUR_API_KEY>",
    ProjectId = "<YOUR_PROJECT_ID>"
});

var response = await client.ListAsyncValidationTaskIssuesAsync(Guid.Parse("88d94fed-4899-4944-9b4b-c919b11a9db0"));
// EndDocSection

File: js/management-api-v2/GetValidationIssues.js
// DocSection: mapi_v2_get_validation_issues
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
// Using ES6 syntax
import { ManagementClient } from '@kontent-ai/management-sdk';

const client = new ManagementClient({
  environmentId: '<YOUR_ENVIRONMENT_ID>',
  apiKey: '<YOUR_API_KEY>'
});

const response = await client.listEnvironmentValidationIssues()
  .byTaskId('15c68e60-dc30-4d34-a921-396563abd08d')
  .toPromise();
// EndDocSection